src/history.js

```diff
@@ -0,0 +1,2 @@
+import { createBrowserHistory } from "history";
+export default createBrowserHistory();
```

Comment:
Delete this file and import  `createBrowserHistory ` where you need that. e.g. `src/index.js`